code/modules/mob/living/simple_animal/bot/vibebot.dm

```diff
@@ -0,0 +1,90 @@
+/mob/living/simple_animal/bot/vibebot
+	name = "\improper vibebot"
+	desc = "A little robot. It's just vibing, doing its thing."
+	icon = 'icons/mob/aibots.dmi'
+	icon_state = "vibebot"
+	density = FALSE
+	anchored = FALSE
+	health = 25
+	maxHealth = 25
+	damage_coeff = list(BRUTE = 1, BURN = 1, TOX = 0, CLONE = 0, STAMINA = 0, OXY = 0)
+	pass_flags = PASSMOB
+
+	radio_key = /obj/item/encryptionkey/headset_service //doesn't have security key
+	radio_channel = RADIO_CHANNEL_SERVICE //Doesn't even use the radio anyway.
+	model = "Vibebot"
+	window_id = "vibebot"
+	window_name = "Discomatic Vibe Bot v1.05"
+	data_hud_type = DATA_HUD_DIAGNOSTIC_BASIC // show jobs
+	path_image_color = "#2cac12"
+
+	var/TurnedOn = FALSE
+	var/current_color
+	var/TimerID
+	var/range = 7
+	var/power = 3
+
+/mob/living/simple_animal/bot/vibebot/Initialize()
+	. = ..()
+	update_icon()
+	auto_patrol = TRUE
+	Vibe()
+
+/mob/living/simple_animal/bot/vibebot/get_controls(mob/user)
+	var/dat
+	dat += hack(user)
+	dat += showpai(user)
+	dat += text({"
```

Comment:
Using text() in anno domini 2020. You're also adding these directly instead of using Join.